src/router/routes.js

```diff
@@ -158,6 +158,13 @@ export default [
 			authenticationRequired: true,
 		}
 	},
+	{
+		path: '/register/guest',
+		component: () => import('@/pages/LoginAndRegister/GuestAccountClaim'),
+		meta: {
+			activeLoginRequired: true,
```

Comment:
The user won't be logged in at this point, so we shouldn't require an active login